Enable CART to run out of the box

blakeflei opened the pull request:
Did my best to enable an out of the box run. I.e. a `make demo` would run if the user doesn't make any changes, and just clicks `Click here to calculate routes`. This didn't work for a few different reasons detailed below (each corresponds to a commit). **These were my best guesses, and may need some tweaking to keep in line with the vision of the project.**

* The compiled OSRM python library bindings couldn't be found by python, so the adjusting the python path environment variable was updated to be an absolute path.
* The `build_parameter.yml` file was expected to be in the same path python is started from, but the container does not start python in the root directory. There may be better solutions that what I've done, but specifying the path to `build_parameter.yml` as one directory level below `local_dir` seems to be resilient to both containerized and host-based runs. It is, however, generally brittle due to being specified in a different location than runs are specified (e.g. will break if another directory layer is added between the data folders and the root directory). This does work for the time being. Is the root directory where `build_parameter.yml` should be located?
* Uploaded files are dropped into the `/data{session_id}` folder, which is not the same as the `WORKING_DATA_DIR` path. Updated the `INPUT_DATA_DIR` to correctly reflect this (I think this is intended behavior given the upload paths created by the streamlit app). Also, for the out of the box (oob) behavior, there is no session id, so `get_solution()` was updated to enable oob runs. 
*`request_solution()` and `download_solution()` were also updated to enable oob runs.


Address [Issue 60](https://github.com/datakind/dk-routing/issues/60).

Zebreu commented:
I will decline it if you confirm it runs out of the box now. A fresh build from main seems to solve all the issues.

Zebreu commented:
Also, I'll avoid touching the input data dirs defaults since this was working fine for the Google Drive integration and I intend to keep it compatible. 
The root directory is the intended location for the build parameters yaml file given it's, in production context, done at build time rather than associated with specific route scenarios.

blakeflei commented:
Thanks so much for providing input and addressing the concerns! 

Running from the dkroutingtool directory addresses the first two points, and [PR 62](https://github.com/datakind/dk-routing/pull/62) addresses the last two points. I'll mark this one closed. Thanks!
